release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -80,6 +80,11 @@ This release introduces the following updates to the *Developer* perspective of
 [id="ocp-4-16-openshift-cli"]
 === OpenShift CLI (oc)
 
+[id="ocp-4-16-oc-adm-upgrade-status-tp"]
+==== Introducing `oc adm upgrade status` command (Tech Preview)
+
+Previously, customers could retrieve information about the status of their cluster update from the web console and from the CLI. In addition to its primary focus, the `oc adm upgrade` command provided some limited information about the upgrade status. With this release, there is now a tech preview `oc adm upgrade status` command which decouples the status information from `oc adm upgrade` and compliments the UI by providing customers with specific information regarding their cluster update, including the status of the control plane and worker node updates.
```

Comment:
This is just a suggestion so up to you, but do you think "user" would be a better word choice than customer? I see user used more often in the release notes and our broader docs, but I think both the ISG and RHSSG say that "customer" is perfectly acceptable to use in a case like this:
```suggestion
Previously, users could retrieve information about the status of their cluster update from the web console and from the CLI. In addition to its primary focus, the `oc adm upgrade` command provided some limited information about the upgrade status. With this release, there is now a tech preview `oc adm upgrade status` command which decouples the status information from `oc adm upgrade` and compliments the UI by providing users with specific information regarding their cluster update, including the status of the control plane and worker node updates.
```